config.template.yml

```diff
@@ -575,6 +580,12 @@ notifier:
     #   --- KEY START
     #   --- KEY END
 
+    ## The lifespans configure the expiration for these token types.
+    # access_token_lifespan: 1h
+    # authorize_code_lifespan: 1h
```

Comment:
I think this should be set to something like 30 seconds by default and in comments . Authorization codes do not need more in my opinion.

Reply:
I've done this. Did 60s, and documented it.